catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -243,7 +247,7 @@ protected Object executeWithSubject() throws Exception {
       start = Instant.now();
       jarSigner.signJar(
           zipFile.getFile(),
-          System.getProperty("org.codice.ddf.system.hostname"),
+          System.getProperty("org.codice.ddf.external.hostname"),
```

Comment:
❓ Should this use the SystemBaseUrl constant?

Reply:
Fixed